### PR TITLE
:sparkles:  Copying and duplicating component tests

### PR DESCRIPTION
**Pull request by superalex**

### CODE REVIEW

File: common/src/app/common/test_helpers/compositions.cljc [hunk 1]
```diff
@@ -35,6 +36,15 @@
                                 :name "Frame1"}
                                params)))
 
+(defn add-group
+  [file group-label & {:keys [] :as params}]
+  ;; Generated shape tree:
+  ;; :group-label [:type :group :name: Group1]
```
Comment (hirunatan) [inline common/src/app/common/test_helpers/compositions.cljc]: Small typo `:name: Group1`

Comment (hirunatan) [reply common/src/app/common/test_helpers/compositions.cljc]: (occurs in other functions too)

File: common/test/common_tests/logic/copying_and_duplicating_test.cljc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+   [app.common.types.container :as ctn]
+   [app.common.types.file :as  ctf]
+   [app.common.uuid :as uuid]
+   [clojure.test :as t]))
+
+(t/use-fixtures :each thi/test-fixture)
+
+(defn- setup []
+  (-> (thf/sample-file :file1)
+      (tho/add-simple-component :simple-1 :frame-simple-1 :rect-simple-1 :child-params {:type :rect :fills (ths/sample-fills-color :fill-color "#2152e5") :name "rect-simple-1"})
```
Comment (hirunatan) [inline common/test/common_tests/logic/copying_and_duplicating_test.cljc]: plz split long lines :)

File: common/src/app/common/test_helpers/compositions.cljc (hunk 1 above)
Comment (superalex) [reply common/src/app/common/test_helpers/compositions.cljc]: Ups!, thanks, fixed! :)

**Pull request merged**